src/jats/jats-exporter.ts

```diff
@@ -2025,29 +2024,52 @@ export class JATSExporter {
       table.insertBefore(tfoot, tbody as Element)
     }
   }
-
+  private moveBody = (body: HTMLElement) => {
+    const container = body.querySelector(':scope > sec[sec-type="body"]')
+    if (!container) {
+      return
+    }
+    const sections = body.querySelectorAll(
```

Comment:
You should be able to query `container` directly: `container.querySelectorAll('> sec')`.